tests/Database/Attach/ResizerTest.php

```diff
@@ -72,6 +72,30 @@ public function testReset()
         $this->assertImageSameAsFixture(self::COMMON_FIXTURES['reset']);
     }
 
+    /**
+     * Given a Resizer with any image
+     * Test to see if the getExtension() method properly returns the filename extension for the provided filename
+     * @throws Exception
```

Comment:
This doesn't seem like it throws an exception does it?

Reply:
@LukeTowers This is what all TestCase test methods throw, isn't it?

Reply:
not if they're written correctly? Generally if a test method expects to throw an exception, it will call $this->expectException(); (or whatever the actual name is).